src/h5web/visualizations/shared/Separator.tsx

```diff
@@ -0,0 +1,9 @@
+import React, { ReactElement } from 'react';
+
+import styles from './Separator.module.css';
+
+function Separator(): ReactElement {
+  return <span className={styles.sep} />;
```

Comment:
Just a dumb component to make it easy to group buttons visually on the toolbar. I think it's important that toggler buttons and button groups are separated from non-toggleable buttons like the screenshot button, to make it clearer that they don't behave the same way. I did make sure they looked slightly different (e.g. no inner shadow on "normal" buttons), but I don't think it's quite enough.